src/content/datachannel/filetransfer/js/main.js

```diff
@@ -15,7 +15,7 @@ var receiveChannel;
 var pcConstraint;
 var bitrateDiv = document.querySelector('div#bitrate');
 var fileInput = document.querySelector('input#fileInput');
-var downloadDiv = document.querySelector('a#received');
+var downloadAnchor = document.querySelector('a#download');
```

Comment:
You need to update the test to look for `a#download` instead of `a#received`:
https://github.com/webrtc/samples/blob/gh-pages/src/content/datachannel/filetransfer/js/test.js#L32 and the text: https://github.com/webrtc/samples/blob/gh-pages/src/content/datachannel/filetransfer/js/test.js#L35